pkg/steps/write_params.go

```diff
@@ -3,6 +3,7 @@ package steps
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/ci-tools/pkg/results"
```

Comment:
nit: import ordering here and in the file above (I might add a linter for that :P)

Reply:
Please do, my editor does this for me and I never look